### DISCUSSION TITLE
Tailwind with Ruby on Rails is using styles for local but looks normal on produciton.

### DISCUSSION THREAD

**Discussion by Osnyder12** [tailwindlabs/tailwindcss#9322] [Help]
Hello. I just started a new rails 7.0.0 app using Tailwind for styling. 

**The Issue**
- When running `./bin/dev` locally  on my laptop, the page is styled incorrectly and looks how it should for mobile (1 grid column)
- On production the page is styled in the correct way with 3 grid columns rather than 1.
- ex. for my homepage i have `class="grid lg:grid-cols-3 md:grid-cols-3 grid-cols"`
- Could this just be my classes? or is there some config i am missing?

**tailwind.config.js**
`const defaultTheme = require('tailwindcss/defaultTheme')

module.exports = {
  content: [
    './public/*.html',
    './app/helpers/**/*.rb',
    './app/javascript/**/*.js',
    './app/views/**/*.{erb,haml,html,slim}'
  ],
  theme: {
    extend: {
      fontFamily: {
        sans: ['Inter var', ...defaultTheme.fontFamily.sans],
      },
    },
  },
  plugins: [
    require('@tailwindcss/forms'),
    require('@tailwindcss/aspect-ratio'),
    require('@tailwindcss/typography'),
  ]
}`

**application.tailwind.css**
`@tailwind base;
@tailwind components;
@tailwind utilities;
`

**Comment by wongjn** [comment]
Is the CSS getting generated correctly? Do you see the class rules in the CSS in dev?

As an aside, `lg:grid-cols-3` is obsolete since you already have `md:grid-cols-3`, and also `grid-cols` is not a valid Tailwind class so unless this is a custom class, this is redundant.
